src/main/java/org/cryptomator/ui/mainwindow/VaultListController.java

```diff
@@ -106,6 +113,19 @@ public class VaultListController implements FxController {
 	public void initialize() {
 		vaultList.setItems(vaults);
 		vaultList.setCellFactory(cellFactory);
+
+		vaultList.prefHeightProperty().bind(vaultList.fixedCellSizeProperty().multiply(vaultList.getItems().size()));
+		vaultList.maxHeightProperty().bind(vaultList.prefHeightProperty());
+		vaultList.prefWidthProperty().bind(scrollPane.widthProperty());
+
+		vbox.heightProperty().addListener((_, oldValue, newValue) -> {
+			if(newValue.doubleValue()>oldValue.doubleValue()){
+				scrollPane.setVvalue(1.0);
+			}
+		});
+
+		vaults.addListener((ListChangeListener<Vault>) _ -> vaultList.prefHeightProperty().bind(vaultList.fixedCellSizeProperty().multiply(vaultList.getItems().size())));
```

Comment:
You're binding the height property to `cellHeight * itemCount` *every time, the vault list changes*.

Either set a binding just once or use a normal setter within the listener. But don't update the binding on every change.